src/detect-flowbits.c

```diff
@@ -243,14 +292,21 @@ int DetectFlowbitSetup (DetectEngineCtx *de_ctx, Signature *s, const char *rawst
     cd = SCMalloc(sizeof(DetectFlowbitsData));
```

Comment:
suggest using `SCCalloc`, this initializes the memory. I think this should fix the scan-build issue below.